src/commands/uncommit.ts

```diff
@@ -42,9 +42,16 @@ export async function _uncommit(parsedSettings: ParsedSettings): Promise<void> {
   const contents = await fsp.readFile(lastMigrationFilepath, "utf8");
   const { headers, body } = parseMigrationText(lastMigrationFilepath, contents);
 
+  // Remove included migrations
+  const includeRegex =
+    /^--![ \t]*Included[ \t]+(?<filename>.*?\.sql)[ \t]*$.*?^--![ \t]*EndIncluded[ \t]*\k<filename>[ \t]*$/gms;
```

Comment:
We don't want newlines to be included in filenames, so I've removed the `/s` flag and been more explicit. I'm actually not sure what our validation is on include paths, whether they're allowed to contain spaces/etc? For now, I've decided a filename is anything but whitespace. I don't think we need to require here that it ends in `.sql`?

```suggestion
    /^--![ \t]*Included[ \t]+(?<filename>\S+)[ \t]*$[\s\S]*?^--![ \t]*EndIncluded[ \t]*\k<filename>[ \t]*$/gm;
```